ecosystem/controllers/runner.py

```diff
@@ -135,3 +135,38 @@ def workload(self) -> Tuple[str, List[CommandExecutionSummary]]:
             self.logger.warning("There in no terra version file...")
 
         return terra_version, [tox_tests_res]
+
+
+class PythonStyleRunner(Runner):
+    """Runners for styling Python repositories."""
+
+    def __init__(self,
+                 repo: Union[str, Repository],
+                 working_directory: Optional[str] = None,
+                 ecosystem_deps: Optional[List[str]] = None,
+                 repo_config: Optional[RepositoryConfiguration] = None):
+        super().__init__(repo=repo,
+                         working_directory=working_directory)
+        self.ecosystem_deps = ecosystem_deps or ["qiskit"]
+        self.repo_config = repo_config
+
+    def workload(self) -> List[CommandExecutionSummary]:
+        """Runs styles checks for python repository.
+
+        Steps:
+        - optional: check for .pylintrc file
+        - run lint
+        - form report
+
+        Returns: execution summary of steps
+        """
+        # check for existing .pylintrc file
+        if os.path.exists(f"{self.cloned_repo_directory}/.pylintrc"):
+            self.logger.info(".pylintrc file exists.")
+            os.rename(f"{self.cloned_repo_directory}/.pylintrc",
+                      f"{self.cloned_repo_directory}/.pylintrc_default")
+
```

Comment:
before running linter we need to create lint file for repo, like we are doing for tox. Otherwise linter will check standard configuration, but we want to take control on that :)

Reply:
oh yes indeed, I need this function too :
```
        # render new tox file for tests
        with open(f"{self.cloned_repo_directory}/tox.ini", "w") as tox_file:
            tox_file.write(repo_config.render_tox_file(
                ecosystem_deps=self.ecosystem_deps))
```